tests/ts_simple_http_api_SUITE.erl

```diff
@@ -211,6 +226,13 @@ select_test(Cfg) ->
         "\"rows\":[[\"q1\",\"w1\",11,110],"
         "[\"q1\",\"w1\",20,119]]}" = Body.
 
+select_with_null_test(Cfg) ->
+    Select = "select * from bob where a='qN' and b='wN' and c>1 and c<99 and d is null",
+    {ok,"200", Headers, Body} = execute_query(Select, Cfg),
+    "application/json" = content_type(Headers),
+    "{\"columns\":[\"a\",\"b\",\"c\",\"d\"],"
+        "\"rows\":[[\"qN\",\"wN\",11,[]]]}" = Body.
```

Comment:
Shouldn't that be `null` instead of `[]`? Such as here: https://github.com/basho/riak_test/pull/1230/files#diff-fab6193d0172e564f9e34f4cda49fefdR358.